src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -444,6 +445,28 @@ export default class HaAutomationActionRow extends LitElement {
             )
           )}
         </ha-md-menu-item>
+        ${type !== "condition"
+          ? html`
+              <ha-md-menu-item
+                .clickAction=${this._onContinueOnError}
+                .disabled=${this.disabled}
+              >
+                <ha-svg-icon
+                  slot="start"
+                  .path=${(this.action as NonConditionAction)
+                    .continue_on_error === true
+                    ? mdiAlertCircleCheck
+                    : mdiAlertCircleCheckOutline}
+                ></ha-svg-icon>
+
+                ${this._renderOverflowLabel(
+                  this.hass.localize(
+                    "ui.panel.config.automation.editor.actions.continue_on_error"
+                  )
+                )}
```

Comment:
I don't feel like with just the icon changing between two very similar variants, and the label staying static, that I would really be able to tell what state this was in. 

A checked/unchecked checkbox would be more intuitive maybe? 

Other toggles also change their verbiage between state e.g. Enable / Disable, but I don't know if "Enable Continue on Error / Disable Continue on Error" is uncomfortably long for the dropdown. 


I would maybe wait for UX review before making any changes though.